docs/regressions-beir-v1.0.0-bioasq-flat.md

```diff
@@ -0,0 +1,68 @@
+# Anserini Regressions: BEIR (v1.0.0) &mdash; bioasq
```

Comment:
Also, @justram - should we go through each of the headings and add the correct capitalization of each corpus? E.g., BioASQ not bioasq?